tls/extensions/s2n_server_key_share.c

```diff
@@ -199,21 +201,27 @@ static int s2n_server_key_share_recv_pq_hybrid(struct s2n_connection *conn, uint
     POSIX_ENSURE(client_kem_group_params->ecc_params.evp_pkey, S2N_ERR_BAD_KEY_SHARE);
     POSIX_ENSURE(client_kem_group_params->kem_group == server_kem_group_params->kem_group, S2N_ERR_BAD_KEY_SHARE);
 
-    uint16_t received_total_share_size;
-    POSIX_GUARD(s2n_stuffer_read_uint16(extension, &received_total_share_size));
-    POSIX_ENSURE(received_total_share_size == server_kem_group_params->kem_group->server_share_size, S2N_ERR_BAD_KEY_SHARE);
-    POSIX_ENSURE(s2n_stuffer_data_available(extension) == received_total_share_size, S2N_ERR_BAD_KEY_SHARE);
+    uint16_t actual_hybrid_share_size;
+    POSIX_GUARD(s2n_stuffer_read_uint16(extension, &actual_hybrid_share_size));
```

Comment:
Nit: since you're touching this anyway...
```suggestion
    uint16_t actual_hybrid_share_size = 0;
    POSIX_GUARD(s2n_stuffer_read_uint16(extension, &actual_hybrid_share_size));
```

Reply:
Fixed